roles/jenkins_common/tasks/main.yml

```diff
@@ -1,28 +1,30 @@
 - name: install beaker client
+  become: true
```

Comment:
I think that become:true as default best suits our use case, where it is needed the large majority of the time.  If you agree with this I can make the necessary modfications to put this in site.yml and re-test.  Let me know.